api/src/main/webapp/WEB-INF/appengine-web.xml.template

```diff
@@ -18,4 +18,8 @@
   <resource-files>
     <include path="WEB-INF/sa-key.json"></include>
   </resource-files>
+
+  <automatic-scaling>
```

Comment:
I really hope we find a way that we don't have to do this is non-prod in future. But for now seems fine.

Reply:
Me too. My hope is that we can enable this in dev for a few weeks, decide whether or not it feels like it helped, then turn it off.